src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -20,7 +23,9 @@ public Integer oddSum(List<Integer> numbers) {
      * Вернуть количество вхождений объекта `element`</p>
      **/
     public Long elementCount(List<String> elements, String element) {
-        return null;
+        return elements.stream()
+                .filter(string -> string.contains(element))
```

Comment:
what is the difference between contains() and equals()?

Reply:
done